### PR TITLE
Create PRCheck.yml

### PR DESCRIPTION
**Pull request by Zaai90**

### CODE REVIEW

File: .github/workflows/PRCheck.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm install eslint-plugin-react@latest --save-dev --force
```
Comment (Zaai90) [inline .github/workflows/PRCheck.yml:29]:
```suggestion
    - run: npm install eslint-plugin-react@latest --save-dev
```

File: .github/workflows/PRCheck.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm install eslint-plugin-react@latest --save-dev
```
Comment (Zaai90) [inline .github/workflows/PRCheck.yml:29]:
```suggestion
```

File: .github/workflows/PRCheck.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm run check
```
Comment (Zaai90) [inline .github/workflows/PRCheck.yml:29]:
```suggestion
    - run: npm install eslint-plugin-react@latest
    - run: npm run check
```

**Review by carlafalk — APPROVED** [review]
wp!

**Review by osci-the-orca — APPROVED** [review]
hehe dumma typescript o svg

**Pull request merged**